trace_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package grpc
+
+import (
+	"testing"
+)
+
+func TestMethodFamily(t *testing.T) {
```

Comment:
`func (s) TestMethodFamily(t *testing.T) {`

This is to run post-test checks for leaked goroutines.
Though it isn't necessary here, this will make `vet` happy.

Reply:
@menghanl Thanks. Will do.